releasenotes/notes/dns-capture.yaml

```diff
@@ -4,5 +4,4 @@ area: networking
 releaseNotes: |
   *Added* experimental DNS resolution directly in the sidecar to better support service entries with TCP services, multicluster DNS resolution,
   and dns resolution of Kubernetes services from VM sidecars. This feature is disabled by default and can be enabled by setting the following
-  in the Istio Operator: meshConfig.defaultConfig.proxyMetadata.ISTIO_META_DNS_CAPTURE="ALL". Setting this option will cause Istio to set the
-  `ndots` option in pod's `dnsConfig` to 4 for faster DNS resolution in common cluster setups.
+  in the Istio Operator: meshConfig.defaultConfig.proxyMetadata.ISTIO_META_DNS_CAPTURE="ALL".
```

Comment:
So we want to mention how to set The new ndots option in release notes?

Reply:
Nope. This is envoy level ndots for use by c-ares library. We are not tinkering with the pod's ndots option at all. In fact ndots 1 is the default everywhere c-ares is used.